roles/ansible-keylime-tpm20/files/tpm2-abrmd.service.d/10-tcti-emulator.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+ExecStart=
```

Comment:
Extra `ExecStart=`

Reply:
It's actually necessary. Directives are additive, so if I didn't have the extra one it would have 2 `ExecStart`s, the original and this one. So to reset it and then set it just the one we want to override we have to first "clear" it out like this.

Reply:
Ah I see, thanks for explaining. I guess I'm not familiar with systemd drop-ins. Can you just append `--tcti=mssim` then? Or is the entire line additive? Is this documented well somewhere? Perhaps we could add a comment somewhere explaining this behavior or linking to a doc that explains it.

Reply:
That's a good point, it's kind of an underused feature. You can see it documented in the very last line of https://www.freedesktop.org/software/systemd/man/systemd.unit.html

And no you can't just append with ExecStart, at least from what I can tell. Systemd does some weird things when you specify multiple ExecStart depending on how you prefix things (see https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStart=).

Reply:
Added a link to the documentation